.github/workflows/tests.yml

```diff
@@ -53,3 +53,20 @@ jobs:
         with:
           args: "--validate"
         if: matrix.python-version == '3.8'
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Setup YFM
+        run: |
+          npm i @doc-tools/docs -g
+          wget https://raw.githubusercontent.com/Toloka/docs/main/.yfm
+      - name: Install stubmaker
```

Comment:
What is the correct writing of this library name? *stubmaker* or *Stubmaker*?

Reply:
Never thought of that. Let it be Stubmaker as it seems that the majority of popular python packages are named starting with the capital letter.